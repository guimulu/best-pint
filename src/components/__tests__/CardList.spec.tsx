import * as React from "react";
import { createSnapshot } from "jest-react-fela";

import { CardList } from "..";

describe("CardList component", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<CardList />);
    expect(snapshot).toMatchSnapshot();
  });
});
