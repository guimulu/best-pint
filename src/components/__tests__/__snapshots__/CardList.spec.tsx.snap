// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList component renders correctly 1`] = `
".a {
  display: flex;
}
.b {
  flex-direction: row;
}
.c {
  flex-wrap: wrap;
}
.d {
  justify-content: space-evenly;
}
.e {
  margin: auto 8px;
}
@media (min-width: 1400px) {
  .f {
    margin: auto 8%;
  }
}


<div className=a b c d e f />;
"
`;
